Update from auto-update/renku-core-2.8.0 (#3385)
SwissDataScienceCenter/renku: CHANGELOG.rst

.. _changelog:

0.45.0
------

Renku ``0.45.0`` introduces improvements when working with projects and sessions
on both the CLI and the UI.

User-Facing Changes
~~~~~~~~~~~~~~~~~~~

**✨ Improvements**

- **Core Service, CLI**: Add support for specifying a project image during
  project initialization

>   (`#3623 <https://github.com/SwissDataScienceCenter/renku-python/issues/3623>`).

Reviewer: ```suggestion
  (`#3623 <https://github.com/SwissDataScienceCenter/renku-python/issues/3623>`_).
```

Reviewer 2: Ooops :speak_no_evil:  Thank you for spotting those!

- **CLI**: Add support for pausing & resuming remote sessions from the cli

>   (`#3633 <https://github.com/SwissDataScienceCenter/renku-python/issues/3633>`).

Reviewer: ```suggestion
  (`#3633 <https://github.com/SwissDataScienceCenter/renku-python/issues/3633>`_).
```

Internal Changes
~~~~~~~~~~~~~~~~

**✨ Improvements**

- **Core Service, CLI**: Make slug and name consistent with rest of platform

>   (`#3620 <https://github.com/SwissDataScienceCenter/renku-python/issues/3620>`).

Reviewer: ```suggestion
  (`#3620 <https://github.com/SwissDataScienceCenter/renku-python/issues/3620>`_).
```

- **Core Service**: Add prometheus metrics

>   (`#3640 <https://github.com/SwissDataScienceCenter/renku-python/issues/3640>`).

Reviewer: ```suggestion
  (`#3640 <https://github.com/SwissDataScienceCenter/renku-python/issues/3640>`_).
```

- **UI**: Adapt dataset APIs to the new naming convention used in the backend

>   (`#2854 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2854>`).

Reviewer: ```suggestion
  (`#2854 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2854>`_).
```

Individual components
~~~~~~~~~~~~~~~~~~~~~

- `renku-ui 3.17.1 <https://github.com/SwissDataScienceCenter/renku-ui/releases/tag/3.17.1>`_

> - `renku-python 2.8.0 <https://github.com/SwissDataScienceCenter/renku-python/tree/v2.8.0>`

Reviewer: ```suggestion
- `renku-python 2.8.0 <https://github.com/SwissDataScienceCenter/renku-python/tree/v2.8.0>`_
```

  (`#1666 <https://github.com/SwissDataScienceCenter/renku-graph/issues/1666>`_).
- **KG**: A new `GET /knowledge-graph/version` API.
  (`#1760 <https://github.com/SwissDataScienceCenter/renku-graph/pull/1760>`_).

> - **Core Service, CLI**: Add support for specifying project images.
>   (`#3623 <https://github.com/SwissDataScienceCenter/renku-python/issues/3623>`)

Reviewer 3: ```suggestion
- **Core Service, CLI**: Add support for specifying a project image during project initialization.
  (`#3623 <https://github.com/SwissDataScienceCenter/renku-python/issues/3623>`)
```

- **CLI**: Add support for pausing/resuming sessions.
  (`#3633 <https://github.com/SwissDataScienceCenter/renku-python/issues/3633>`)
- **Core Service**: Add prometheus metrics.

>   (`#3640 <https://github.com/SwissDataScienceCenter/renku-python/issues/3640>`)

Reviewer 3: Please move this to the "Internal Changes" section
